### ISSUE TITLE
DateTime error in the dashboard

### ISSUE THREAD

**Issue by Sodki** [TheJumpCloud/jumpcloud-dashboard#16]
When running the dashboard, I get the following error:

```
New-UDPage : Exception calling "Invoke" with "0" argument(s): "Exception calling "Invoke" with "0" argument(s): "Cannot convert value "13/01/2020 11:51:37" to type "System.DateTime". Error: "String '13/01/2020 11:51:37' was not recognized as a valid DateTime."""
At /home/hr/.local/share/powershell/Modules/JumpCloud.Dashboard/1.0.2/Private/Content-Pages/Default/2Get-UDSystems.ps1:17 char:15
+     $UDPage = New-UDPage -Name:($PageName) -Content {
+               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : SyntaxError: (UniversalDashboard.Models.Page:Page) [New-UDPage], CmdletInvocationException
+ FullyQualifiedErrorId : UniversalDashboard.Cmdlets.NewPageCommand
```

I'm running the dashboard on Ubuntu 19.10, PowerShell 6.2.4 and using the locale `en_GB.UTF-8`.

**Comment by kmoorehead-jc** [comment]
Hey @Sodki!

Thanks for bringing this to our attention and for including all your system information.

I'll be doing some reproduction, fixing and testing on this issue today. I'll make sure to keep you in the loop and hope to have an update for you soon!

**Comment by kmoorehead-jc** [comment]
Heloo @Sodki!

I just wanted to take a second to update you on this issue. We were able to track this down to an issue in the JumpCloud PowerShell module relating to DateTime in various locales that bled over to the Dashboard.

The fix for this has been pushed in v1.15.4 of the JumpCloud PowerShell Module. Once you update that and restart your session, you should no longer see this error.

I'll leave this open for a few days to make sure that you still aren't experiencing issues.

Have a good day!

**Comment by Sodki** [comment]
Hi,

I don't get any errors anymore, I'm happy to close this ticket.

Thank you so much
